### PR TITLE
Feature/bolt sticky rebased

### PR DESCRIPTION
**Pull request by remydenton**
Replaces #197.

Please reference comments from that PR and push any updates to this new rebased branch.

### CODE REVIEW

File: src/_patterns/02-components/bolt-sticky/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+## Getting Started
+The best way to start using Bolt is via the *Bolt Starterkit* (coming soon), a pre-assembled front-end boilerplate designed to get you up and running as quickly as possible.
+
+You can also use the full [Bolt Core](https://www.npmjs.com/package/@bolt/core) CSS framework on its own, or, install just the parts you need and fold the different ITCSS layers of Bolt into your existing codebase.
+
+Installing Bolt’s Action Blocks component is as easy as running NPM install:
+
+```
+npm install @bolt/sticky
```
Comment (sghoweri) [inline src/_patterns/02-components/bolt-sticky/README.md]: Shouldn't this be `npm install @bolt/components-sticky`?

File: src/_patterns/02-components/bolt-sticky/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# bolt/components-sticky
+Sticky makes any component sticky to the top of the page. Part of the Bolt “Components” CSS framework that powers the [Bolt Design System](https://www.boltdesignsystem.com).
+
+## Demo
+
+```
+<div class="c-bolt-sticky">
```
Comment (sghoweri) [inline src/_patterns/02-components/bolt-sticky/README.md]: @remydenton you should update this to include the Twig embed so the template stays in sync:

```
{% embed "@bolt/sticky.twig" %}
  {% block sticky_content %}
    <div style="background-color: lightblue">
      First sticky item
    </div>
  {% endblock %}
{% endembed %}
```

File: src/_patterns/02-components/bolt-sticky/package.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    {
+      "name": "Salem Ghoweri",
+      "email": "me@salemghoweri.com",
+      "web": "https://github.com/sghoweri"
+    },
+    {
+      "name": "Mike Mai",
+      "email": "boss@mikemai.net",
+      "web": "http://mikemai.net/"
+    }
```
Comment (sghoweri) [inline src/_patterns/02-components/bolt-sticky/package.json]: @remydenton you should add your maintainer info here!

File: src/_patterns/02-components/bolt-sticky/src/sticky.scss [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  display: block;
+}
+
+.c-bolt-sticky {
+  display: block;
+  position: sticky;
+  top: 0;
+  width: 100%;
+  z-index: 99999; // TODO: [Mai] We need z-index management.
+}
```
Comment (sghoweri) [inline src/_patterns/02-components/bolt-sticky/src/sticky.scss]: Missing empty line at end of file

Comment (remydenton) [reply src/_patterns/02-components/bolt-sticky/src/sticky.scss]: Looks to me like it already has a newline actually?  If a newline is missing, you'll typically see this in github:

File: src/_patterns/02-components/bolt-sticky/README.md (hunk 2 above)
Comment (remydenton) [reply src/_patterns/02-components/bolt-sticky/README.md]: I updated the docs to at least use an embed.  I'm a little uncertain about the content though-- isn't the demo content itself intended to be throwaway?  I assume this part of the docs is just intended to be something that someone can use to start as quickly as possible and see how it's desiged to work.

File: src/_patterns/02-components/bolt-sticky/src/sticky.scss (hunk 4 above)
Comment (sghoweri) [reply src/_patterns/02-components/bolt-sticky/src/sticky.scss]: You're totally right - missed that. Thanks!

File: src/_patterns/02-components/bolt-sticky/README.md (hunk 2 above)
Comment (sghoweri) [reply src/_patterns/02-components/bolt-sticky/README.md]: Totally (that demo content was just an example) - I just wanted to make sure we're clearly communicating that embedding this (vs hard coding some HTML) is the recommended way to wire this up (same thing applies to  other wrapper-like components)

**Comment by sghoweri** [comment]
Merged (but keeping this branch around till you guys say we can delete it) - thanks @remydenton @mikemai2awesome! I'll get this published first thing tomorrow morning!

**Comment by remydenton** [comment]
@sghoweri, I think we should make a habit of deleting branches once they're merged to keep down on the clutter.

The easiest way IMO is to this the 'Delete Branch' button on the bottom of a github PR.  It even gives  you a handy 'Restore' button after you delete should you need the branch back for any reason.

**Pull request merged**